crawler/attribution/entrypoints/eventlistener.js

```diff
@@ -0,0 +1,9 @@
+{
+    const img = document.createElement('img');
+    img.src = '/';
+    const src = `${document.currentScript.src}`;
+    img.addEventListener('error', () => {
+        console.log(src, navigator.userAgent);
```

Comment:
Add result from inside the listener?

Is a regular function any different also?

Reply:
I just mean engine wise they're quite different so might diverge in implementation.

Happy to ignore also.

Reply:
Right now it doesn't make a difference on the crawling results. If it does in the future we will need to cover both cases, but for now I think we can leave it as is